Remotion/ObjectBinding/Web.UnitTests/UI/Controls/BocListImplementation/Rendering/BocColumnRendererArrayBuilderTest.cs

```diff
@@ -57,7 +57,22 @@ public void GetColumnRenderers ()
     }
 
     [Test]
-    public void GetColumnRenderers_WithBocValidationErrorIndicatorColumnDefinition_AlwaysVisibility_WithoutFailures_ContainsNullColumnRenderer ()
+    public void CreateColumnRenderers_WithNotVisibleColumn_CreatesNullColumnRenderer ()
```

Comment:
please add either a second test or do it here:
one column before and one after. 
Please add assertion for VisibleColumnIndex to all column items.